flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskCancelAsyncProducerConsumerITCase.java

```diff
@@ -58,7 +60,8 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
-public class TaskCancelAsyncProducerConsumerITCase extends TestLogger {
+@ExtendWith({TestLoggerExtension.class})
```

Comment:
Isn't it possible to set this up for all classes automatically?

Reply:
Yes, but I want this PR to focus on the job about changing rules to extensions. I change this test to show how to use  `AllCallbackWrapper`. And I will raise other PRs to migrate modules to JUnit5.